shuffle-yarn/pom.xml

```diff
@@ -185,7 +192,7 @@ under the License.
 		<dependency>
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-examples-batch_${scala.binary.version}</artifactId>
-			<version>${flink.version}</version>
+			<version>${rpc.flink.version}</version>
```

Comment:
In subsequent versions, will we keep using this RPC version?

Reply:
Yes, this help to keep compatibility of client and server RPC.